script/coin-price-data-fetcher/config/default.js

```diff
@@ -0,0 +1,38 @@
+module.exports  = {
+  sourceCurrencies: ['ADA', 'ERG'],
+  targetFiatCurrencies: ['USD', 'JPY', 'EUR', 'CNY', 'KRW', 'BRL'],
```

Comment:
Can we add CAD and IDR? We have a considerable amount of users on mobile.

Reply:
We'll add it into backlog